jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
+        self.spin_counter = 50
+        self.current_round = 1
+        self.turnPoints = 0
+        self.question_bank
+
+    'Add a player to the list'
+    def addPlayer(self, name):
+        Player = player(name)
+        self.player_list.append(Player)
+        return
+
+    'Remove a player from the list'
+    def deletePlayer(self, player):
+        self.player_list.remove(player)
+        return
+
+    'Get the next player in the list'
+    def nextPlayer(self):
+        len = self.player_list.count
+        current = self.current_player
+
+        if (current+1 >= len):
+            self.current_player = 0
+        else:
+            self.current_player = current+1
+
+        self.resetTurnPoints()
+        return
+
+    'Get the current player'
+    def getCurrentPlayer(self):
+        current = self.current_player
+        return self.player_list(current)
+
+    'Reset the points for current turn'
+    def resetTurnPoints(self):
+        self.turnPoints = 0
+        return
+
+    """
+    Update the points for the current turn
+    based on the point value and question result.
+    If the answer was correct (true) add points, else subtract
+    """
+    def updateTurnPoints(self, points, result):
+        turnPoints = self.calculateRoundPoints(points)
+
+        if (result):
+            self.turnPoints += turnPoints
+        else:
+            self.turnPoints -= turnPoints
+        return
+
+    'Calculate points based on the round'
+    def calculateRoundPoints(self, points):
+        return (points * self.current_round)
+
+    'Update the current player score'
+    def updatePlayerScore(self):
```

Comment:
This call all be shortened. No need to store these temporary variable. Good job though.

Reply:
Would it work if I just called self.getCurrentPlayer().set_player_score(points) ? Would Python be able to resolve that?